tests/unit/array/test_dataclass.py

```diff
@@ -0,0 +1,60 @@
+from typing import Optional, Union
+
+import pytest
+
+from docarray import Document
+from docarray.typing import Text, Image, Audio, JSON
+from docarray.dataclasses.types import _is_optional, dataclass, _is_optional_field
+
+
+def test_dataclass_opregetional_value():
+    @dataclass
+    class MultiModalDoc:
+        foo: Text
+
+    d = Document(MultiModalDoc(foo='sfdssf'))
+
+
+def test_dataclass_optional_value():
```

Comment:
why `bar`? where does this bar come from?

Reply:
my bad it should be foo. This test is not passing atm anyway

Reply:
adjust PR description